Client/python_client.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+import requests
+import os
+from Client import status
+
+
+class PythonClient:
+    """
+    This class represents the python client
+    """
+    def __init__(self):
+        self.url = 'http://127.0.0.1:5000/'
+
+    def send_file(self, path: str, email: str = None) -> str:
+        """
+        This function sends a file to the server
+        :param path: the path of the file
+        :param email: optional email to attach to the file
+        :return: the uid of the file
+        """
+
+        with open(path, 'rb') as file:
+            if email:
+                response = requests.post(self.url, files={'file': file}, data={'email': email})
+            else:
+                response = requests.post(self.url, files={'file': file})
+
+        if response.status_code == 200:
+            return response.json()['uid']
+        else:
+            raise Exception(response.text)
```

Comment:
Always throw the most specific exception -
https://stackoverflow.com/questions/2052390/manually-raising-throwing-an-exception-in-python
https://docs.python.org/3/library/exceptions.html

Reply:
Fix everywhere